site/app/Resources/views/notification/notification_list.html.twig

```diff
@@ -0,0 +1,33 @@
+{# Render Notifications #}
+{% if mgilet_notification_unseen_count(app.user) != 0 %}
+    {% for notification in notificationList %}
+        {% if notification.seen == false %}
```

Comment:
You should be able to combine the for and if by using `filter` (https://twig.symfony.com/doc/2.x/filters/filter.html).